tests/core/policies/test_intent_ted_policy.py

```diff
@@ -0,0 +1,380 @@
+from tests.core.policies.test_ted_policy import TestTEDPolicy
+from pathlib import Path
+from typing import Optional, Dict, List
+import tensorflow as tf
+
+import numpy as np
+import pytest
+from _pytest.monkeypatch import MonkeyPatch
+from rasa.core.featurizers.single_state_featurizer import (
+    IntentTokenizerSingleStateFeaturizer,
+)
+from rasa.core.featurizers.tracker_featurizers import (
+    IntentMaxHistoryTrackerFeaturizer,
```

Comment:
I noticed that this `IntentMaxHistoryTrackerFeaturizer` doesn't exist. Is it something that's done separately?

Reply:
Yes, that's being implemented [here](https://github.com/RasaHQ/rasa/pull/8570)